### ISSUE TITLE
Move Table for ops to TD spec

### ISSUE THREAD

**Issue by mmccool** [w3c/wot-architecture#606]
Propose cleaning up ops table and move to TD document.

- [ ] move ops table to TD spec (eg. PR to delete here, PR to add there)
- [ ] edit existing definitions to be consistent (e.g. Property vs. Property Affordance, etc)

See also https://github.com/w3c/wot-profile/pull/92

If you object to moving, pls comment below...

**Comment by mmccool** [comment]
More generally ops etc. are related to a discussion about definitions of what actions, events, and properties are anyway (standard collections of resources?  *categories* of resources) and what ops are (standard methods?) and how these relate to RESTful APIs etc.  Some of this belongs in architecture, but this is proposing that the detailed, specific table belongs in the TD spec because that is where people will be looking for definitions of these specific enumeration values.

**Comment by egekorkan** [comment]
Is there any blockers regarding creating a PR in the TD spec?

**Comment by mlagally** [comment]
@egekorkan 
I agree that we should avoid duplication where it makes sense or potentially causes divergence.

Note that we have a clear structure and split of concerns between the architecture and TD spec since we released the 1.0 version of the spec.

The architecture spec contains terminology, application domains, concepts, interaction model, deployment scenarios, introduction to building blocks, guidelines and security guidelines. 
The thing description contains the data model and a JSON-LD serialisation.

We should avoid breaking this structure if we don't have to. As we agreed in today's editors call, the usual reader starts reading the architecture and dive deeper into corresponding building blocks / specifications. This may be the TD, discovery, scripting or binding templates. 

It is obvious that there are some overlaps, specifically when it comes to these operations. These have been described in the architecture spec for the last 3 years and provide a good overview of the interaction semantics.

I completely agree that the section in the TD spec that describes operations requires significant additional work and a more detailed technical description. However I would think that the level of description has to be more detailed than just adding a simple table without describing concrete semantics of each operation, interaction flows and error behavior.

I would expect that the TD specification includes a subsection for each of these operations with these details.

**Comment by egekorkan** [comment]
@mlagally wrote: 
> Note that we have a clear structure and split of concerns between the architecture and TD spec since we released the 1.0 version of the spec.

It seems that this is actually not true. See  #625, #619, #412, https://github.com/w3c/wot-discovery/issues/190. My guess is that even the editors of other specs do not read the normative parts of the architecture document, resulting in collisions.

> However I would think that the level of description has to be more detailed than just adding a simple table without describing concrete semantics of each operation, interaction flows and error behavior.
I would expect that the TD specification includes a subsection for each of these operations with these details.

This would be actually in the architecture document if such a table describing operations is also here. Explanation of such behavior has nothing to do with the TD document. Opening an issue here about this.

**Comment by mlagally** [comment]
arch call on 25.11.: definitions of operations are in the TD, the concepts should be explained and some replacement text should be proposed here. See also: https://github.com/w3c/wot-architecture/issues/626

**Comment by egekorkan** [comment]
Not sure what I should do here
